### PR TITLE
Cli updates for indexes + reputation module

### LINKED ISSUE #84: Integrate index and reputation apis into cli

### PR DESCRIPTION
**Pull request by asutula**
Easy way to get some feedback about fil-tools working or not

Closes #84

### CODE REVIEW

**Comment by asutula** [comment]
This also removes asks-related code from the deals module. That all now exists in the asks index service and asks cli command

File: exe/fil-tools/cmd/reputation_top_ miners.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package cmd
```
Comment (jsign) [inline exe/fil-tools/cmd/reputation_top_ miners.go]: Seems that this filename contains a space.

Comment (asutula) [reply exe/fil-tools/cmd/reputation_top_ miners.go]: thx, fixed

**Comment by asutula** [comment]
@jsign I'm not sure why the tests are failing. Something in the `OpenCL Dependencies` step. Any ideas?

**Comment by asutula** [comment]
It does indeed look like the file http://azure.archive.ubuntu.com/ubuntu/pool/main/m/mesa/libgbm1_19.2.1-1ubuntu1~18.04.1_amd64.deb doesn't exist, but I'm not sure why it's looking for files that don't exist on that server.

**Comment by jsign** [comment]
Yeah, today seems that the Azure repo that GH action uses got some problem because started failing.
Change the rule to this:
```
      - name: Install OpenCL
        run: sudo apt-get update && sudo apt-get install mesa-opencl-icd ocl-icd-opencl-dev
```
I fixed it today in my branch and it works now.

**Comment by asutula** [comment]
Cool thanks @jsign that worked.

**Pull request merged**